openmc/summary.py

```diff
@@ -64,6 +71,26 @@ def nuclides(self):
         return self._nuclides
 
     @property
+    def fast_materials(self):
```

Comment:
I'm not a fan of the `fast_` names here, but I see the problem that we already have a `materials` attribute so removing `fast_` would conflict with that. What do you think of only keeping these attributes private for now? (i.e., removing these public properties and use `_fast_*` directly in other methods)